src/Components/AllCandidates/CommitteeReview.jsx

```diff
@@ -78,11 +88,23 @@ const Wrapper = styled.div`
 //   return false
 // }
 
+function convertToTableData(fetched) {
+  const committeeList = [];
+  if (fetched !== null) {
+    fetched.forEach((member) => {
+      committeeList.push({
+        committeeMember: member.member.email,
+        candidatesReviewed: member.review,
+      });
+    });
+  }
+  return committeeList;
+}
+
 class CommitteeReview extends Component {
```

Comment:
@ahmedhamodi Is it possible to turn this into a functional component?
I'll end up doing in a future PR if it doesn't happen here anyway, so no rush.

Reply:
Good point, I can do that.

Reply:
Definitely! We can also pair during work session if you want -- no need to do it in this PR if it's blocking any other work 👍 💯

Reply:
I would appreciate that, we can make some time on Monday to go through this